lib/services.rb

```diff
@@ -37,6 +37,10 @@ def self.elasticsearch(hosts: ENV['ELASTICSEARCH_HOSTS'] || 'http://localhost:92
   def self.statsd_client
     @statsd_client ||= Statsd.new.tap { |sd| sd.namespace = "govuk.app.rummager" }
   end
+
+  def self.worldwide_api
+    @worldwide_api ||= GdsApi::Worldwide.new(Plek.new.find('whitehall-admin'))
```

Comment:
How about `GdsApi.worldwide` – that way if the common way to connect to this API changes we inherit those changes too.

Reply:
I get "undefined method `worldwide' for GdsApi:Module`" when making that change. Any idea why?

Reply:
Hmm, it should work 🤔 

https://github.com/alphagov/gds-api-adapters/blob/c9cae38a7b5d96837bc64c853420566f2668617b/lib/gds_api.rb#L181-L183

What version of the gem are you using?

Reply:
Have you required the right files?

Reply:
I think so. Could it be because GdsApi is a module defined by Rummager, too?

Reply:
That would be it yeh! 🤦‍♂️